bindings/python/iota_sdk/client/responses.py

```diff
@@ -18,70 +18,118 @@
 from iota_sdk.types.transaction_id import TransactionId
 from iota_sdk.types.transaction_metadata import TransactionFailureReason, TransactionState
 
+# Node routes responses
+
 
 @json
 @dataclass
-class OutputIdsResponse:
-    """Response type for output IDs.
+class RoutesResponse:
+    """API route groups of the node.
+    GET /api/routes.
 
     Attributes:
-        committed_slot: The committed slot at which these outputs were available at.
-        page_size: The maximum amount of items returned in one call. If there are more items, a cursor to the next page is returned too.
-        cursor: The cursor to the next page of results.
-        items: The query results.
+        routes: The available API route groups of the node.
     """
+    routes: List[str]
 
-    def __init__(self, output_dict: Dict):
-        self.committed_slot = output_dict["committedSlot"]
-        self.page_size = output_dict["pageSize"]
-        self.cursor = output_dict["cursor"]
-        self.items = [OutputId.from_string(
-            output_id) for output_id in output_dict["items"]]
+
+@json
+@dataclass
+class ProtocolParametersResponse:
+    """Protocol Parameters with start epoch.
+
+    Attributes:
+        start_epoch: The start epoch of the set of protocol parameters.
+        parameters: The protocol parameters.
+    """
+    start_epoch: EpochIndex
+    parameters: ProtocolParameters
 
 
 @json
 @dataclass
-class CommitteeMember:
-    """Information of a committee member.
+class InfoResponse:
+    """General information about the node.
+    GET /api/core/v3/info.
 
     Attributes:
-        address: Account address of the validator.
-        pool_stake: The total stake of the pool, including delegators.
-        validator_stake: The stake of a validator.
-        fixed_cost: The fixed cost of the validator, which it receives as part of its Mana rewards.
+        name: The name of the node (e.g. Hornet).
+        version: The semantic version of the node.
+        status: The status of the node.
+        metrics: Node metrics.
+        protocol_parameters: Supported protocol versions by the node.
+        base_token: Gives info about the base token the network uses.
     """
-    address: str
-    pool_stake: int = field(metadata=config(
-        encoder=str
-    ))
-    validator_stake: int = field(metadata=config(
+    name: str
+    version: str
+    status: NodeInfoStatus
+    metrics: NodeInfoMetrics
+    protocol_parameters: List[ProtocolParametersResponse]
+    base_token: NodeInfoBaseToken
+
+
+@json
+@dataclass
+class NodeInfoResponse:
+    """General information about the node and its URL.
+    GET /api/core/v3/info.
+
+    Attributes:
+        info: A NodeInfo object.
+        url: The URL of the node.
+    """
+    info: InfoResponse
+    url: str
+
+# Accounts routes responses
+
```

Comment:
```suggestion
```

Reply:
sorry, fancy python formatter says we absolutely need this blank line :trollface:

Reply:
oh, turns out that if there are 2 blank lines above the `#` then it's fine to only have one below. I guess you're fine with this, so I gnna do it locally.